### DISCUSSION TITLE
How can I get out of inline code ?

### DISCUSSION THREAD

**Discussion by wenqvip** [outline/outline#2557] [Self Hosting] [answered]
somethings like this `inline code `
how can I get out of the code block and start input normal text. I can't use left/right arrow keys to move the cursor out of the inline code and all inputing texts are become part of the inline code.

**Comment by tommoor** [comment, marked as answer]
Arrow keys should work, which version are you using?

**Reply by wenqvip** [reply]
I figure it out. It seems that I should press right arrow firstly, then press space, then will find the cursor moves out of the inline code.
Thank you anyway :)

**Reply by wenqvip** [reply]
I encounter another problem: how to get out of block code ?
```
some code
```

**Reply by tommoor** [reply]
Hit `enter` twice or arrow down if there is other content below
